Twig/Extension/FormatterMediaExtension.php

```diff
@@ -76,14 +76,6 @@ public function getTwigExtension()
     }
 
     /**
-     * {@inheritdoc}
-     */
-    public function getName()
```

Comment:
The problem is the  `FormatterMediaExtension` class which extends `BaseProxyExtension`